### PR TITLE
Feat: API calls

### PR DESCRIPTION
**Pull request by nishasen**
Added all the API calls required for fetching note and archive

### CODE REVIEW

File: wiznotes/src/APICalls/PostArchive.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+export const PostArchive = async(note, archiveDispatch, noteDispatch) => {
+    console.log(note)
+    try{ 
+        const config = {
+            headers: {
+                authorization: localStorage.getItem("userToken"),
+            }
+        };
+        const response = await axios.post(`/api/notes/archives/${note._id}`, { note }, config);
+        console.log(response)
```
Comment (sadathshariff) [inline wiznotes/src/APICalls/PostArchive.js]: remove console.log

**Review by sadathshariff — APPROVED** [review]
Looks good, liked how you have broken it into small functions

**Review by tanishq20 — APPROVED** [review]
Very optimized code. Like the way how you have abstracted all the API calls.

File: wiznotes/src/APICalls/PostArchive.js (hunk 1 above)
Comment (nishasen) [reply wiznotes/src/APICalls/PostArchive.js]: yes okay

**Pull request merged**